xml/sec_images_hardened.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+
+<sect1 xml:id="sec-images-hardened" xml:lang="en"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ 
+ <title>Hardened Images</title>
+ <para>
+  To improve overall security, &suse; provides hardened images of some products.
+  The images are hardened using <phrase role="product">&openscap;</phrase>, a
+  collection of open source tools that implement the <systemitem
+  class="protocol">Security Content Automation Protocol (SCAP)</systemitem>
+  maintained by the <orgname>National Institute of Standards and Technology
+  (NIST)</orgname>. <phrase role="product">&openscap;</phrase> supports
+  automated configuration, vulnerability and patch checking, technical control
+  compliance activities, and security measurement.
+ </para>
+ <para>
+  All images are pre-hardened to the extent they can safely be hardened without
+  causing problems in public cloud frameworks. Certain rules can only be
+  applied after instance creation, for example:
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    Rules that require having passwords set up. Passwords are not yet configured
+    when images are built.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Rules that affect the network configuration. Networking is set up during
+    instance creation, therefor it is not yet possible to limit access.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Rules for custom partitioning. &suse;'s public cloud images use a standard
+    partitioning with a single root file system. You can create additional
+    partitions after instance creation.
```

Comment:
There's nothing "standard" about partitioning, we need to be careful in using this word. In addition in Azure we have a separate boot partition as such the "single root file system" statement is not true. I suggest we write in this way.

"""
Rules for custom partitioning. &suse;'s public cloud images are partitioned to meet the requirements of the framework in which they are released. For systems that need to adhere to standards that require separate file systems for given directories we recommend that you build your own images and use LVM or move those directories onto attached disks to get the strictest data separation possible.
"""